### PR TITLE
Consolidate move commands into a single function

### PR DESCRIPTION
**Pull request by OFurnell**

### CODE REVIEW

File: src/game.js [hunk 1]
```diff
@@ -358,10 +357,7 @@ load(
 			map.render();
 			mapText.render();
 			player.render();
-			enemies.forEach(function(enemy){
```
Comment (duff9) [inline src/game.js:361]: why the change here?

Comment (OFurnell) [reply src/game.js:361]: Consistency with the line below it. Not really needed but I snuck it in
